test/routes/actions.test.js

```diff
@@ -20,6 +20,40 @@ function initContext(schema) {
   return context;
 }
 
+async function callLoadHook(load) {
+  const schemas = {
+    users: {
+      actions: [{
+        name: 'send invoice',
+        hooks: { load },
+        fields: [{ field: 'invoice number', type: 'String' }],
+      }],
+    },
+  };
+  const {
+    actions, model, app, logger,
+  } = initContext(schemas).inject();
+
+  const request = { body: { data: { attributes: { recordsId: [1] } } } };
```

Comment:
As said previously, I think it will be:
`const request = { body:  { recordsId: [1] } };`

Reply:
Fixed.